dexml/__init__.py

```diff
@@ -272,6 +272,7 @@ def __init__(self,**kwds):
         Keyword arguments that correspond to declared fields are processed
         and assigned to that field.
         """
+        self.meta = copy.copy(self.__class__.meta)
```

Comment:
An alternative to a `copy` here, might be to create an object that allows modifications local to the instance, but falls back to the class for keys it doesn't know about.  This would mean that any changes made on the class still affect all its instances.  What do you think?